content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -153,6 +153,34 @@ section refers to several key workload enablers and how they map to Windows.
   * `emptyDir` volumes
   * Named pipe host mounts
   * Resource limits
+  * OS field: `pod.spec.os.name` should be set to `windows` to indicate that the current pod is a Windows pod. 
+              Please note that there is no effect when setting a value for this field since it is behind a feature gate `IdentifyPodOS` and it is currently alpha.
```

Comment:
if the feature gate is enabled does it have an affect? It seems from below it does. Is this field available even when the feature gate is not set? If that is the case, it would be clearer to say something along those lines.

Reply:
can we use the feature-state functionality here for consistency and also to avoid using 'currently' (which may get stale)
{{< feature-state for_k8s_version="v1.23" state="alpha" >}}

Ex: https://github.com/kubernetes/website/blob/269dad43a0d6d7df31bfc9d7b08faeaa866a4f78/content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md?plain=1#L608-L610

Reply:
Usually all the fields will be available irrespective of the featuregates. We try to drop fields when persisting to storage so that it won't have any effect. 

More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#adding-unstable-features-to-stable-versions. Look at the create and update strategy code.

Reply:
What does no effect mean here? This says there is no effect but then goes on to say "the following fields must be unset on the Windows pods when the `IdentifyPodOS` featuregate is enabled" which implies there is a difference in behavior: if you set this field then your pod will be rejected if those fields are set.

Reply:
> What does no effect mean here?

OS field has no effect when the `IdentifyPodOS` featuregate is disabled. It can be `windows` or `linux` but it doesn't matter however when the `IdentifyPodOS` featuregate is enabled and if linux specific fields are set in the security context of the pod when the name is set to `windows`, pod validation would fail

Reply:
It needs to be clearer that there is no affect (besides the older kubectl version that @sftim mentioned below)  when disabled but when enabled there is an affect (validation could fail). Right now this reads as there is not affect for this field.

Reply:
left a possible suggestion